distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso

```diff
@@ -417,7 +417,7 @@ type Invalid_Value_Type
     to_display_text self =
         prefix = case self.related_column of
             Nothing -> "Expected "
-            column_name -> "Expected " + column_name + " column to have "
+            column_name : Text -> "Expected " + column_name + " column to have "
```

Comment:
Can we add a catch all here please. I know we shouldn't get here but throwing an error in the `to_display_text` will mean we just can't see anything in IDE if something set wrongly. (e.g. we passed a Column instead of the name)